package.json

```diff
@@ -23,15 +23,24 @@
     "ethereum-waffle": "^3.4.0",
     "ethers": "^5.4.7",
     "hardhat": "^2.6.5",
+    "hardhat-contract-sizer": "^2.5.1",
     "hardhat-deploy": "^0.10.5",
     "hardhat-deploy-ethers": "^0.3.0-beta.10",
     "hardhat-gas-reporter": "^1.0.4",
     "ts-node": "^10.2.1",
-    "typescript": "^4.4.3",
-    "hardhat-contract-sizer":"^2.5.1"
+    "typescript": "^4.4.3"
   },
   "dependencies": {
     "@openzeppelin/contracts-upgradeable": "4.5.2",
+    "@prb/math": "^2.5.0",
     "solidity-coverage": "^0.7.20"
+  },
+  "prettier": {
```

Comment:
Is this from another repo? How did we come up with these values?

Reply:
I adapted this from TreasureProject/treasure-infra (but changed line length to 120). But I can remove this just to minimize changes.